testrunner-running/src/main/scala/edu/illinois/cs/statecapture/CustomReflectionConverter.java

```diff
@@ -0,0 +1,57 @@
+package edu.illinois.cs.statecapture;
+
+import com.thoughtworks.xstream.converters.ConversionException;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter;
+import com.thoughtworks.xstream.converters.reflection.ReflectionConverter;
+import com.thoughtworks.xstream.converters.reflection.ReflectionProvider;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.mapper.Mapper;
+
+import java.lang.reflect.Field;
+
+public class CustomReflectionConverter extends ReflectionConverter {
```

Comment:
Should this extend that CustomAbstractReflectionConverter? They seem to have the same logic in the overridden methods.